modules/ROOT/pages/message-logging-policy.adoc

```diff
@@ -3,53 +3,55 @@ ifndef::env-site,env-github[]
-|Level |Defines if the message is logged according to the configuration in your log4j2 file.
-|Before Calling API |Set to cause your logging policy to be applied before the API is called.
-|After Calling API |Set to cause your logging policy to be applied after the API is called.
+>s| Policy Name | Message Logging
+>s|Summary      | Logs custom messages using information from incoming requests, the backend, or information from other policies applied to the same API endpoint.
+>s|Category | Troubleshooting
+>s|First Mule version available | 3.8.0
+.1+>.^s| Returned Status Codes
+|None
 |===
 
-== About the Logging Format
-
-A log has the following format: `<date> <thread name> <level> <category>: <message>`
+The Message Logging policy logs custom messages using information from incoming requests, the response from the backend, or information from other policies applied to the same API endpoint. 
 
-category org.mule and message #[payload] (where payload value is "the payload") is:
+This policy works with both Apache log4j and Apache log4j2.
 
-`2018-01-25 18:57:29,907 [WrapperListener_start_runner] INFO org.mule: the payload`
+== Prerequisites
 
+You must have the Anypoint Platform organization administrator (OrgAdmin) role or have permissions to create or manage APIs.
 
-== About Policy Implementation
+== How This Policy Works
 
 Logging a policy consists of the following sequence:
 
 . A request is sent to the API.
-. The message is logged, if the following are true:
-+
+. The message is logged, if the following conditions are true:
 ** The level and category of the log defined in the policy are included in the logger defined in the log4j2.xml configuration file.
-** The conditional field of the policy was not set, or was set, and the condition was satisfied.
+** The conditional field of the policy was not set, or was set, and the condition was met.
 . The API response is returned. Message Logging Policy does not interfere with the execution of any policy nor its flow.
 . The message appears in the application log.
 
-== About Logging States
+== About the Logging Format
+
+A message log uses the following format: 
++
+`<date> <thread name> <level> <category>: <message>`
++
+`date`: The date and timestamp when the message is logged.
+`thread name`: The event thread for which the message is logged.
+`level`: The logging level or type of the message, which is the same as log4j levels.
++
+For more information, see https://www.tutorialspoint.com/log4j/log4j_logging_levels.htm[log4j Logging Levels].
+`category`: This parameter value that typically indicates the package and class where the logging occurred. The default value is `org.mule`.
+`message`: This parameter displays the details of the type of issue, for example, "the payload".
++
+Example:
+`2018-01-25 18:57:29,907 [WrapperListener_start_runner] INFO org.mule: the payload`
+
+== Understanding the Logging States
```

Comment:
```suggestion
== Logging States
```
...but consider something more precise: "When Message Logging Occurs" or "Events That Cause Message Logging" or similar.

Reply:
Logging States During Events